tests/compiler/memory.untouched.wat

```diff
@@ -4,25 +4,25 @@
  (type $i32_i32_i32_i32_=>_none (func (param i32 i32 i32 i32)))
  (import "env" "abort" (func $~lib/builtins/abort (param i32 i32 i32 i32)))
  (memory $0 1)
- (data (i32.const 16) "\00\00\00\00")
- (data (i32.const 28) ",\00\00\00\01\00\00\00\00\00\00\00\01\00\00\00\12\00\00\00m\00e\00m\00o\00r\00y\00.\00t\00s\00\00\00\00\00\00\00\00\00\00\00")
- (data (i32.const 80) "\00")
- (data (i32.const 96) "\00")
- (data (i32.const 104) "\00")
- (data (i32.const 108) "\00")
- (data (i32.const 110) "\00")
- (data (i32.const 111) "\00")
- (data (i32.const 112) "\00")
- (data (i32.const 128) "\00")
- (data (i32.const 144) "\00")
- (data (i32.const 152) "\00")
- (data (i32.const 156) "\00")
- (data (i32.const 158) "\00")
- (data (i32.const 159) "\00")
- (data (i32.const 160) "\00")
+ (data (i32.const 16) "")
+ (data (i32.const 28) ",\00\00\00\01\00\00\00\00\00\00\00\01\00\00\00\12\00\00\00m\00e\00m\00o\00r\00y\00.\00t\00s")
+ (data (i32.const 80) "")
+ (data (i32.const 96) "")
+ (data (i32.const 104) "")
+ (data (i32.const 108) "")
+ (data (i32.const 110) "")
+ (data (i32.const 111) "")
+ (data (i32.const 112) "")
+ (data (i32.const 128) "")
+ (data (i32.const 144) "")
+ (data (i32.const 152) "")
+ (data (i32.const 156) "")
+ (data (i32.const 158) "")
+ (data (i32.const 159) "")
+ (data (i32.const 160) "")
```

Comment:
Could such empty data segments also avoided after trimming?

Reply:
Hmm, good question. In this case we'd also lose the visual info that there is a data segment starting there. Still unsure how useful this or the zeroes are when manually viewing compiler output, somewhat depending on how common it is to run untouched binaries, and how far we'd like to go in terms of exactly displaying what becomes compiled.

Reply:
> Hmm, good question. In this case we'd also lose the visual info that there is a data segment starting there

Good point